assets/component-product-model.css

```diff
@@ -10,7 +10,7 @@
 }
 
 .product__xr-button[data-shopify-xr-hidden] {
-  visibility: hidden;
+  display: none;
```

Comment:
cc @melissaperreault Can I get your 👍  on this, just to confirm that we changed our approach from previous product media decisions?

------

**For context:** The `visibility:hidden` was intentionally set to avoid content jumping while the page is loading. This follows our approach on every previous theme that had 3D media implemented. We decided (at the time) that having some extra whitespace was a smaller issue than this page jumping:

<img src="https://user-images.githubusercontent.com/37168033/134057768-af74148e-bd64-426d-b324-2aac6710235a.gif" width="300">

This can be an issue on slower connections (simulate `Slow 3G connection` on Chrome Developer Tools to better understand this), where the page might still be loading and the user starts interacting with it, then the `View in your space` button appears and pushes the content - possibly making the user click a button by accident.

Reply:
At first glance I am concerned by this page jump happening without user interaction and can lead to a poor UX. It is something we debated on our internal implementation of our Debut theme. (https://github.com/Shopify/debut/pull/1166)

I would like to bring @tylerrowsell and  @sainihas to the conversation as this is affecting a global feature experience and want to bring the right people in the conversation.

I'd  like to check and sync about other alternatives we could have.

cc. @nicklepine

Reply:
From the technical side of things, one thing that comes to mind as a possible solution to explore... We could detect user agents server side and then make that info available in a liquid drop. It would require a list of all mobile user agents but perhaps could be a starting point. Then we could use progressive enhancements in js to further check ones that we don't detect via user agent. 

I am not confident in how reliable that would be though, for that reason I would suggest we consider an alternative button/layout for "view in your space". 

I'll defer to Sai on this one about potential alternative for design.

Reply:
From the technical side of things, one thing that comes to mind as a possible solution to explore... We could detect user agents server side and then make that info available in a liquid drop. It would require a list of all mobile user agents but perhaps could be a starting point. Then we could use progressive enhancements in js to further check ones that we don't detect via user agent.

I am not confident in how reliable that would be though, for that reason I would suggest we consider an alternative button/layout for "view in your space".

I'll defer to Sai on this one about potential alternative for design.

Reply:
Here is the commit: https://github.com/Shopify/dawn/pull/655/commits/87d531ab6fef175e04859db48675e61e94556a56